src/PointwiseFunctions/AnalyticSolutions/Elasticity/BentBeam.hpp

```diff
@@ -0,0 +1,126 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <cstddef>
+#include <limits>
+
+#include "DataStructures/DataBox/Prefixes.hpp"   // IWYU pragma: keep
+#include "DataStructures/Tensor/Tensor.hpp"      // IWYU pragma: keep
+#include "Elliptic/Systems/Elasticity/Tags.hpp"  // IWYU pragma: keep
+#include "Options/Options.hpp"
+#include "PointwiseFunctions/Elasticity/ConstitutiveRelations/IsotropicHomogeneous.hpp"
+#include "PointwiseFunctions/Elasticity/ConstitutiveRelations/Tags.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+/// \cond
+class DataVector;
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace Elasticity {
+namespace Solutions {
+/*!
+ * \brief A state of pure bending of an elastic beam in 2D
+ *
+ * \details This solution describes a 2D slice through an elastic beam of length
```

Comment:
It would be nice to have the analytic solution here in the form of an equation :+1:

Reply:
you're right, that's an oversight on my part